static/js/dialog_widget.js

```diff
@@ -48,17 +48,15 @@ export function hide_dialog_spinner() {
 }
 
 export function show_dialog_spinner() {
+    if (!settings_data.using_dark_theme()) {
+        $(".dialog_submit_button .loader").attr("src", "/static/images/loader-black.svg");
+    } else {
+        $(".dialog_submit_button .loader").attr("src", "/static/images/loader.svg");
+    }
     $(".dialog_submit_button .loader").css("display", "inline-block");
     $(".dialog_submit_button span").hide();
```

Comment:
Should we move this function to `loading.ts`?  I guess I don't have a great sense for why we can't use the existing functions in that module for these loading spinners, but I think a `loading.show_button_spinner(elt)` interface would avoid duplication here.

Reply:
The disabled look for both button is different - in case the button is changed to gray and in one it is not. We can deduplicate even then but the code will not be clear and much readable I think.

Reply:
OK, makes sense.